packages/console/src/pages/TenantSettings/Subscription/index.tsx

```diff
@@ -43,7 +47,15 @@ function Subscription() {
       <SwitchPlanActionBar
         currentSubscriptionPlanId={currentSubscription.planId}
         subscriptionPlans={reservedPlans}
-        onSubscriptionUpdated={onCurrentSubscriptionUpdated}
+        onSubscriptionUpdated={() => {
+          /**
+           * The upcoming billing info is calculated based on the current subscription usage,
+           * and the usage is based on the current subscription plan,
+           * need to manually trigger the usage update while the subscription plan is changed.
+           */
+          onCurrentSubscriptionUpdated();
+          void mutateSubscriptionUsage();
```

Comment:
Change the function to async and "await" for `mutateSubscriptionUsage`?

Reply:
Good suggestion, this could bring an even better experience.